config/BUILD.bazel

```diff
@@ -1,5 +1,23 @@
 package(default_visibility = ["//visibility:public"])
 
+sh_binary(
```

Comment:
Why not use the go_binary method I suggested? Does embed not work in a different package?

Reply:
gazelle doesn't like it and will wipe off everything :-( this is suggested by @ixdy

Reply:
er, what I meant when I suggested `alias` is to use only `alias`.

i.e., do
```python
alias(
    name = "mkpj",
    actual = "//prow/cmd/mkpj:mkpj",
)
```
here, and then add the args and data attributes back to the rule in //prow/cmd/mkpj.

Reply:
hummm, the mkpj target in //prow/cmd/mkpj should not contain the actual cncf config stuff though (per previous discussion above)